app/src/main/java/com/example/englishappforkid/presentation/screens/home/PreHomeScreen.kt

```diff
@@ -0,0 +1,309 @@
+package com.example.englishappforkid.presentation.screens.home
+
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxHeight
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.width
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Refresh
+import androidx.compose.material3.Button
+import androidx.compose.material3.ButtonDefaults
+import androidx.compose.material3.Card
+import androidx.compose.material3.CardDefaults
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.NavigationBar
+import androidx.compose.material3.NavigationBarItem
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
+import com.example.englishappforkid.R
+import com.example.englishappforkid.ui.theme.boxBackground
+import com.example.englishappforkid.ui.theme.boxFullname
+import com.example.englishappforkid.ui.theme.colorButtonBarBackGround
+import com.example.englishappforkid.ui.theme.colorButtonSelected
+import com.example.englishappforkid.ui.theme.icRefresh
+
+@Composable
+fun preHomeScreen() {
+    Column(
+        modifier =
+            Modifier
+                .fillMaxSize()
+                .background(Color(0xFFFFFFFF))
+                .padding(24.dp),
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        headerSection()
+        Spacer(modifier = Modifier.height(36.dp))
+
+        menuCard(
+            title = "Short Story",
```

Comment:
```suggestion
            title = stringResource(id =R.string.short_story),
```
update những chỗ khác để sau này mình support đa ngôn ngữ